templates/dashboard/group/list.html

```diff
@@ -0,0 +1,59 @@
+{% extends "dashboard/base.html" %}
+{% load i18n %}
+{% load prices_i18n %}
+{% load materializecss %}
+{% load staticfiles %}
+
+{% block title %}
+  {% trans "Groups" context "Dashboard groups list" %} - {{ block.super }}
+{% endblock %}
+
+{% block body_class %}body-customers{% endblock %}
+
+{% block menu_groups_class %}active{% endblock %}
+
+{% block header_extra %}
+    <div class="fixed-action-btn">
+      <a class="btn-fab btn-floating btn-large waves-effect waves-light teal"
+        title="{% trans "Add group" context "Category list add button text" %}"
+        href="{% url 'dashboard:group-create' %}">
+        <svg data-src="{% static "dashboard/images/add.svg" %}" />
+      </a>
+    </div>
+{% endblock %}
+
+{% block breadcrumbs %}
+  <ul class="breadcrumbs">
+    <li class="visible-s">
+      {% trans "Groups" context "Dashboard groups list" %}
+    </li>
+  </ul>
+{% endblock %}
+
+{% block content %}
+  {% if groups %}
+    <ul class="collection list">
+      {% for group in groups %}
+        <li class="collection-item list-item">
+          <a class="list-item-name" href="{% url 'dashboard:group-details' pk=group.pk %}">
+            {{ group.name }}
+            <p class="list-item-desc">
+              <span>
+
```

Comment:
This span should be empty?

Reply:
Here i want a nice list of permissions but forgot to update it.